src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -209,63 +209,68 @@ def load_arguments(self, _):
         c.argument('key', help='If no key specified, return all keys by default. Support star sign as filters, for instance abc* means keys with abc as prefix.')
         c.argument('label', help="If no label specified, list all labels. Support star sign as filters, for instance abc* means labels with abc as prefix. Use '\\0' for null label.")
 
+    with self.argument_context('appconfig feature') as c:
+        c.argument('key', validator=validate_feature_key, help='Key of the feature flag. Default key is the reserved prefix ".appconfig.featureflag/" + feature name.')
```

Comment:
>help [](http://example.com/codeflow?start=58&length=4)

Can we mentioned that key needs to start with the prefix ".appconfig.featureflag/"